### DISCUSSION TITLE
After accessing cloudflare zero trust and cdn, sonixd can not connect to the server

### DISCUSSION THREAD

**Discussion by agedcat** [navidrome/navidrome#2234] [Q&A]
After accessing cloudflare zero trust and cdn, sonixd can not connect to the server.
I set additional Authentication throught zero trust，maybe it is the reason. How can i fix it?

**Comment by scofieldpeng** [comment]
I deploy navidrome behind cloudflare zero trust and without any additional auth, everything is ok.

navidrome must login and why u set additional auth, do you have any special reason?

**Comment by dhanadhan** [comment]
I am also using the CF Zero Trust and no issue here. 
Likely, it's the additional authentication you added.
